Moq.AutoMocker.TestGenerator/AnalyzerReleases.Unshipped.md

```diff
@@ -0,0 +1,7 @@
+﻿### New Rules
```

Comment:
This will need to get copied into the Shipped.md file on release.

Reply:
What is this then, a separate PR just to update the notes?